src/client.cpp

```diff
@@ -274,6 +270,7 @@ Client::Client(
 	// Add local player
 	m_env.addPlayer(new LocalPlayer(this, playername));
 
+	m_mapper = new Mapper(device, this);
```

Comment:
I don't really like Mapper function. It looks like if we map something to another :s